bugsnag-android-core/src/main/java/com/bugsnag/android/EventStore.java

```diff
@@ -23,7 +23,7 @@
  */
 class EventStore extends FileStore {
 
-    private static final String STARTUP_CRASH = "_startupcrash";
+    private static final String STARTUP_CRASH = "startupcrash";
```

Comment:
Does the presence or not of the underscore affect decoding of API key or event types? I notice that until now, the "not-jvm" part doesn't have an underscore. This changes seems good to make it consistent, but could it affect decoding the information at all?

I notice the javadoc (L222) needs updating for new format.

Reply:
`isLaunchCrashReport` checks for the presence of `_startupcrash.json` and `not-jvm` is not explicitly checked anywhere. The underscore has been added in the `String.format` call for greater consistency so this hsouldn't affect decoding info.

Javadoc updated.